### ISSUE TITLE
Staging sources: prospecting just stay at "drilling" if "you are too hungry" in process

### ISSUE THREAD

**Issue by elfl0rd** [StrangeLoopGames/EcoIssues#13641]
No errors...
Just "drilling"
You can stay in this for hours.
![изображение](https://user-images.githubusercontent.com/4980243/68545831-eacc3d80-03e1-11ea-88d2-5bb681870785.png)

**Comment by Quadbro** [comment]
but it wont actually prospect further?

**Comment by elfl0rd** [comment]
Yes, it stop at position.
But as i think - need something like "error, you are too hungry to do that" and set drilling status to "stop" or "interrupted"

**Comment by SlayksWood** [comment]
I have notification only from third time:
- I start drill, my calorie is out. I stopped drill, but I haven't notification.
- I Hold LMB second time, animation start but stopped immediately. I still haven't notification.
- I hold LMB third time and notification appears.
Notification should appear when I stopped drill first time.
![2020 02 13 prospecting notification only from third time](https://user-images.githubusercontent.com/45708377/74406076-ba13fe00-4e3f-11ea-8105-1eadf3fb7e38.gif)
